lib/workers/repository/dependency-dashboard.spec.ts

```diff
@@ -144,6 +144,7 @@ describe('workers/repository/dependency-dashboard', () => {
     it('open or update Dependency Dashboard when all branches are closed and dependencyDashboardAutoclose is false', async () => {
       const branches: BranchConfig[] = [];
       config.dependencyDashboard = true;
+      config.dependencyDashboardHeader = 'And this is a header';
```

Comment:
I must admit I'm wearing my pedantic hat, but don't you agree it's a bit weird to start a sentence/dashboard message with the word "And "?

Reply:
@ademar59 Add a real sample here, so it will show what you like to achieve

Reply:
done in fix: review